data/json/items/resources/wood.json

```diff
@@ -91,13 +91,13 @@
     "name": "plank",
     "description": "A narrow, thick plank of wood, like a 2 by 4 or similar piece of dimensional lumber.  Makes a decent melee weapon, and can be used for all kinds construction.",
     "category": "spare_parts",
-    "weight": "1391 g",
+    "weight": "1400 g",
```

Comment:
There's no need to revert this but in the future, generally I don't think there's a ton of value to changing things by such tiny amounts. It's not as though this change really added a lot of accuracy, a 2x4x32 inch piece of lumber could still weigh the original weight.

Reply:
I definitely agree there was no need to change 1391 to 1400 except my personal liking for round numbers when they're available.